### ISSUE TITLE
[lit-next] @lit-labs/scoped-registry-mixin

### ISSUE THREAD

**Issue by sync** [lit/lit#1728]
# Description
Exports points to a development scoped-registry-mixin.js but it's missing from published npm package.

```
"exports": {
    ".": {
      "development": "./development/scoped-registry-mixin.js",
      "default": "./scoped-registry-mixin.js"
    }
  },
````

# @lit-labs/scoped-registry-mixin
1.0.0-pre.1

# Compilation error

Could not load /Users/anthonymittaz/Projects/DesignSystems/mono/node_modules/@lit-labs/scoped-registry-mixin/development/scoped-registry-mixin.j

**Comment by sync** [comment]
Looking at the packages.json it looks like `files` is ignoring it.

```
  "files": [
    "/src/",
    "!/src/test/",
    "/scoped-registry-mixin.{d.ts,d.ts.map,js,js.map}"
  ],
```

**Comment by justinfagnani** [comment]
Fixed by #1729
